lib/js/src/protocol/enums/_ControlFrameTags.js

```diff
@@ -72,6 +72,12 @@ _ControlFrameTags.RPC = Object.freeze({
         VIDEO_SERVICE_TRANSPORTS: 'videoServiceTransports',
         /** Auth token to be used for log in into services **/
         AUTH_TOKEN: 'authToken',
+        VEHICLE_MAKE: 'make',
+        VEHICLE_MODEL: 'model',
+        VEHICLE_MODEL_YEAR: 'modelYear',
```

Comment:
It seems this is just a typo in the proposal, on testing alongside the core we received `modelYear` thus included it in camelcase instead of space-separated.